### PR TITLE
Update bandit workflow

### PR DESCRIPTION
**Pull request by sesorov**
### Details:
 - Use new bandit configuration in Actions.

### Tickets:
 - CVS-82553

### CODE REVIEW

File: .github/workflows/bandit.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+      - 'docs/**'
+      - '.ci/azure/**'
+  pull_request:
+    paths:
+      - 'src/bindings/python/**'
+      - 'samples/python/**'
+      - 'tools/**'
+      - 'scripts/utils/**'
+      - 'docs/**'
+      - '.ci/azure/**'
```
Comment (asomsiko) [inline .github/workflows/bandit.yml:20]: Should we exclude folders in single place in bandit config file and run this action on any *.py file? (like ./**/*.py)

File: .github/workflows/bandit.yml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+jobs:
+  bandit_check:
+    runs-on: ubuntu-18.04
```
Comment (asomsiko) [inline .github/workflows/bandit.yml:23]: if possible we should use ubuntu 20 because ubutnu 18 LTS support will end soon.

**Review by asomsiko — CHANGES_REQUESTED** [review]
Bandit check should pass

File: .github/workflows/bandit.yml (hunk 1 above)
Comment (sesorov) [reply .github/workflows/bandit.yml]: Changed this in last commit to run Bandit only when python scripts are updated

File: .github/workflows/bandit.yml (hunk 2 above)
Comment (sesorov) [reply .github/workflows/bandit.yml]: Fixed in last commit

**Comment by sesorov** [comment]
@asomsiko Bandit now green (see example [here](https://github.com/sesorov/openvino/runs/5825349710?check_suite_focus=true))

**Comment by akladiev** [comment]
This PR will be closed in 2 weeks in case of no activity.

**Comment by akladiev** [comment]
This PR was closed because it has been stalled for 2 week with no activity.

**Pull request closed without merging**